SocialCareCaseViewerApi.Tests/V1/Controllers/SocialCareCaseViewerApiControllerTests.cs

```diff
@@ -114,6 +116,49 @@ public void GetPersonByIdReturns200WhenSuccessful()
             response?.StatusCode.Should().Be(200);
         }
 
+        [Test]
+        public void GetPersonByIdDoesNotCallTheCreateRequestAuditUseCaseWhenAuditingIsEnabledIsFalse()
+        {
+            var request = new GetPersonRequest();
```

Comment:
Similar thought as above.

Reply:
Good point, I've updated the tests and added new ones for backwards compatibility check as well.